io.ente.auth.yml

```diff
@@ -0,0 +1,123 @@
+app-id: io.ente.auth
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: ente_auth
+finish-args:
+  # X11 + XShm access  
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  # Flutter needs GPU permission.
+  - --device=dri
+  # Needs to talk to the network for authentication
+  - --share=network
+  # For desktop notifications
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: libayatana-appindicator
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DENABLE_BINDINGS_MONO=NO
+      - -DENABLE_BINDINGS_VALA=NO
+      - -DENABLE_GTKDOC=NO
+    sources:
+      - type: git
+        url: https://github.com/AyatanaIndicators/libayatana-appindicator.git
+        tag: 0.5.93
+        commit: 238c8b02718fa5b4af95ede72beeed762094f4cc
+        x-checker-data:
+          type: git
+    modules:
+      - shared-modules/intltool/intltool-0.51.json
+      - name: ayatana-ido
+        buildsystem: cmake-ninja
+        sources:
+          - type: git
+            url: https://github.com/AyatanaIndicators/ayatana-ido.git
+            tag: 0.10.4
+      - name: libayatana-indicator
+        buildsystem: cmake-ninja
+        config-opts:
+          - -DENABLE_LOADER=NO
+        sources:
+          - type: git
+            url: https://github.com/AyatanaIndicators/libayatana-indicator.git
+            tag: 0.9.4
+            commit: 611bb384b73fa6311777ba4c41381a06f5b99dad
+            x-checker-data:
+              type: git
+      - name: libdbusmenu-gtk3
+        buildsystem: autotools
+        build-options:
+          cflags: -Wno-error
+        config-opts:
+          - --with-gtk=3
+          - --disable-dumper
+          - --disable-static
+          - --disable-gtk-doc
+          - --enable-introspection=no
+          - --disable-vala
+        cleanup:
+          - /share/gtk-doc
+          - /share/libdbusmenu
+        sources:
+          - type: archive
+            url: https://launchpad.net/libdbusmenu/16.04/16.04.0/+download/libdbusmenu-16.04.0.tar.gz
+            sha256: b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a
+
+  - shared-modules/libsecret/libsecret.json
+
+  - name: webkit2gtk-4.0
```

Comment:
what needs webkit2gtk4.0 ?

Reply:
ente_auth needs it to display webview for various pages like Privacy Policy, etc.

Reply:
then use the GNOME runtie. I asked a question, not (yet) requested to change it.

Reply:
gnome runtime dropped webkit2gtk 4.0 in 44.

Reply:
Yes as @bbhtt said, currently the app uses 4.0 so gnome runtime 46 didn't made sense.

Reply:
ok.